projects/aca-playwright-shared/src/page-objects/components/actions-dropdown.component.ts

```diff
@@ -50,9 +50,10 @@ export enum ActionType {
 }
 
 export class ActionsDropdownComponent extends BaseComponent {
-  private static rootElement = 'aca-rule-action-list';
+  private static rootElement = 'mat-dialog-container aca-rule-action-list';
```

Comment:
this is relying on material classes that might go away soon, can you somehow use test id instead?